pages/help.py

```diff
@@ -489,10 +488,9 @@ def get_content(self):
 aware of and are working on.
 """)),
                     FaqItem("kaspersky", _("I have Kaspersky anti-virus, and Freenet doesn't install, or shows \"Download/upload queue database corrupted!\""), _("""
-                    Kaspersky can be a problem with Freenet. See [here](
-https://wiki.freenetproject.org/Installing/Windows#.27Download
-.2Fupload_queue_database_corrupted.21.27_.28When_using_Kaspersky_on_Windows_7
-.29). We recommend you turn off Kaspersky during install and during node
+                    Kaspersky can be a problem with Freenet.
+See [here](https://wiki.freenetproject.org/Installing/Windows#.27Download.2Fupload_queue_database_corrupted.21.27_.28When_using_Kaspersky_on_Windows_7.29).
```

Comment:
According to [their own overview][0], there only is a now-obsolete unmaintained implementation of CommonMark in Python. Apart from that, I'd sure like a somewhat stricter Markdown processor.

As for this and the other links, this was intended as a more ad-hoc solution but now you ask for it, I'll take care of that as well.

[0]: https://github.com/jgm/CommonMark/wiki/List-of-CommonMark-Implementations